addons/website/static/src/snippets/s_table_of_content/000.js

```diff
@@ -40,10 +42,12 @@ const TableOfContent = publicWidget.Widget.extend({
         _.each($fixedElements, el => position += $(el).outerHeight());
         const isHorizontalNavbar = this.$target.hasClass('s_table_of_content_horizontal_navbar');
         this.$target.css('top', isHorizontalNavbar ? position : '');
-        this.$target.find('.s_table_of_content_navbar').css('top', isHorizontalNavbar ? '' : position + 20);
-        const $mainNavBar = $('#oe_main_menu_navbar');
-        position += $mainNavBar.length ? $mainNavBar.outerHeight() : 0;
+        this.tableOfContentNavbarEl.style.top = isHorizontalNavbar ? '' : `${position + 20}px`;
+        const mainNavBarEl = document.body.classList.contains('o_fullscreen') || this.editableMode ? null
+            : document.getElementById('oe_main_menu_navbar');
+        position += mainNavBarEl ? mainNavBarEl.offsetHeight : 0;
         position += isHorizontalNavbar ? this.$target.outerHeight() : 0;
+        this.tableOfContentNavbarEl.style.maxHeight = isHorizontalNavbar ? '' : `calc(100vh - ${position + 40}px`;
```

Comment:
Is not this missing a parenthesis to close the `calc` ? :thinking:

Reply:
@qsm-odoo This is ready if you want to add it in your todo list ;)